fairmq/shmem/UnmanagedRegion.h

```diff
@@ -294,6 +321,30 @@ struct UnmanagedRegion
         }
     }
 
+    void InitializeRefCountSegment(uint64_t size)
+    {
+        using namespace boost::interprocess;
+        if (!fRefCountSegment) {
+            fRefCountSegment = std::make_unique<managed_shared_memory>(open_or_create, fRefCountSegmentName.c_str(), size);
+            LOG(trace) << "shmem: initialized ref count segment: " << fRefCountSegmentName;
+            fRefCountPool = std::make_unique<RefCountPool>(fRefCountSegment->get_segment_manager());
+        }
+    }
+
+    RefCount* GetRefCountAddressFromHandle(const boost::interprocess::managed_shared_memory::handle_t handle)
+    {
+        if (fRefCountPool) {
+            return reinterpret_cast<RefCount*>(fRefCountSegment->get_address_from_handle(handle));
+        } else {
+            return nullptr;
+        }
```

Comment:
```suggestion
        if (fRefCountPool) {
            return reinterpret_cast<RefCount*>(fRefCountSegment->get_address_from_handle(handle));
        }
        return nullptr;
```

see [readability-else-after-return](https://clang.llvm.org/extra/clang-tidy/checks/readability/else-after-return.html)